be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/handlers/LaoHandler.scala

```diff
@@ -35,7 +35,7 @@ case object LaoHandler extends MessageHandler {
         // Publish on the LAO main channel
         dbActor.ask(ref => DbActorNew.Write(channel, rpcMessage.getParamsMessage.get, ref)) match {
           case Success(_) => Left(rpcMessage)
-          case _ => Right(PipelineError(-10, "")) // FIXME add DbActor "answers" with error description if failed
+          case _ => Right(PipelineError(-10, "")) // FIXME add DbActor "answers" with error description if failed. Should be a SERVER_ERROR
```

Comment:
`ErrorCodes.SERVER_ERROR` here please. Very likely the clients won't give any feedback on seeing an unknown error code.

Reply:
Yep! Also added to my todo list :) Same as the last PR. I unfortunately need to wait for everything to be in a single PR to resolve that